doc/user-manual/tools/interface-files.rst

```diff
@@ -0,0 +1,54 @@
+.. _interface-files:
+
+********************
+Interface files
+********************
+
+.. note::
+   This is a stub. Contributions, additions and corrections are greatly
+   appreciated.
+
+Probably you already noticed that when you save an agda file, another file
```

Comment:
Are we addressing the reader directly in any other part of the manual?
That reads a bit strange.

Reply:
Changed. Sorry, I have to get used to the formal writing style that is used in the docs.